lib/language.js

```diff
@@ -5844,6 +5844,7 @@ function init() {
     ,'Start of record validity' : {
       cs: 'Začátek platnosti záznamu'
       ,ro: 'De când este valabilă înregistrarea'
+      ,de: Starte Aufzeichnungsgültigkeit'
       ,sv: 'Starttid för händelse'
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2469782181/issues/source?bid=2452379&fileBranchId=3259564#l5848)